search_events_app/tests/models/test_country.py

```diff
@@ -16,3 +21,33 @@ def test_country_basic_info(self):
 
     def test_country_str(self):
         self.assertEqual(self.country.__str__(), 'Argentina')
+
+    @patch.object(
+		Country,
+		'objects'
+	)
```

Comment:
Paths could be in one line for more readability

Reply:
In this case, for example, just put
`@patch.object(Country,	'objects')`

Reply:
Resolved